test/behaviour/reasoner/verification/ForwardChainingMaterialiser.java

```diff
@@ -183,13 +181,15 @@ private Materialisations() {
 
         private FunctionalIterator<Materialisation> forConcludable(BoundConcludable boundConcludable) {
             FunctionalIterator<Materialisation> materialisations = empty();
-            Optional<Thing> inferredConcept = boundConcludable.inferredConcept();
-            if (inferredConcept.isPresent()) {
-                materialisations = materialisations.link(forThing(inferredConcept.get()));
-            }
+
             Optional<Pair<Thing, Attribute>> inferredHas = boundConcludable.inferredHas();
             if (inferredHas.isPresent()) {
                 materialisations = materialisations.link(forHas(inferredHas.get().first(), inferredHas.get().second()));
+            }else{
```

Comment:
returned the attribute materialisation in addition to the "has" materialisation when a "has" concludable was passed